README.md

```diff
@@ -33,8 +33,8 @@ unzip eva-sub-cli.zip && mv eva-sub-cli-* eva-sub-cli
 Then install the library and its dependencies as follows (e.g. in a virtual environment):
 ```bash
 cd eva-sub-cli
-pip install -r requirements.txt
-python setup.py install
+# Activate your virtual environment 
+python -m pip install .
```

Comment:
I pulled your branch and tried to install, but it doesn't install the dependencies:
```
april@boudica:~/projects/eva-sub-cli$ python3 -m venv test
april@boudica:~/projects/eva-sub-cli$ source ./test/bin/activate
(test) april@boudica:~/projects/eva-sub-cli$ python -m pip install .
Processing /home/april/projects/eva-sub-cli
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
    Preparing wheel metadata ... done
Building wheels for collected packages: eva-sub-cli
  Building wheel for eva-sub-cli (PEP 517) ... done
  Created wheel for eva-sub-cli: filename=eva_sub_cli-0.4.dev36+g1592d35.d20240718-py3-none-any.whl size=268077 sha256=bcabebc0707a34a39908019958aede78488d8b023cdcf2d0d8923e6dac3ace2e
  Stored in directory: /home/april/.cache/pip/wheels/5a/f4/30/0e4664674590f68182eb9b911492e1bb33271cbe92917f5303
Successfully built eva-sub-cli
Installing collected packages: eva-sub-cli
  Attempting uninstall: eva-sub-cli
    Found existing installation: eva-sub-cli 0.4.dev36+g1592d35
    Not uninstalling eva-sub-cli at /home/april/projects/eva-sub-cli, outside environment /home/april/projects/eva-sub-cli/test
    Can't uninstall 'eva-sub-cli'. No files were found to uninstall.
Successfully installed eva-sub-cli-0.4.dev36+g1592d35.d20240718
(test) april@boudica:~/projects/eva-sub-cli$ eva-sub-cli.py --version
Traceback (most recent call last):
  File "/home/april/projects/eva-sub-cli/test/bin/eva-sub-cli.py", line 5, in <module>
    from eva_sub_cli.executables.cli import main
  File "/home/april/projects/eva-sub-cli/test/lib/python3.8/site-packages/eva_sub_cli/executables/cli.py", line 14, in <module>
    from ebi_eva_common_pyutils.logger import logging_config
ModuleNotFoundError: No module named 'ebi_eva_common_pyutils'
(test) april@boudica:~/projects/eva-sub-cli$
```
It works fine if I pip install requirements separately.

Reply:
Should be addressed now. Thanks for testing it.

Reply:
all working now :+1: